### ISSUE TITLE
Docker - Environment Variables not working

### ISSUE THREAD

**Issue by woodjme** [davidnewhall/unifi-poller#146]
With the latest release the environment variables passed in to the container no longer seem to be being used. In this case the poller is trying to login to `127.0.0.1` despite `UP_UNIFI_URL` being present.

`docker inspect unifi-poller`
```
            "Env": [
                ...
                "UP_UNIFI_URL=https://unifi.jamiewood.io",
                "UP_POLLING_INTERVAL=1m",
                "PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin"
            ],
```

`docker logs unifi-poller`
```
2019/12/02 10:12:03 [INFO] UniFi Poller v1.6.0-553 Starting Up! PID: 1
2019/12/02 10:12:03 [ERROR] unifi controller: Post https://127.0.0.1:8443/api/login: dial tcp 127.0.0.1:8443: connect: connection refused
```

**Comment by davidnewhall** [comment]
Sorry, some of them were renamed to make maintenance easier. See here: https://github.com/davidnewhall/unifi-poller/wiki/Docker#environment-variables

**Comment by davidnewhall** [comment]
...but that appears broken. Will look into it!

**Comment by davidnewhall** [comment]
Yup, I managed to break this entirely while trying to make it easier to maintain. Shortly I'll put a fix in `master` branch - pulling the Docker `latest` tag will have the fix. We've identified a few things that need to be fixed in 1.6 so a 1.6.1 will likely be out later this week. Sorry about that!!

**Comment by davidnewhall** [comment]
1.6.1 fixes this, but 1.6.2 is coming, so maybe just wait..
